Exiled.API/Features/Map.cs

```diff
@@ -239,6 +240,69 @@ public static void SpawnHands(Vector3 position, Quaternion rotation)
+        /// <param name="addToChosenList">If set to true, the player will be added to the chosen list and cannot be automatically selected for the remainder of the round.</param>
+        public static void ForceSpawn2536(Player player, bool addToChosenList = true)
+        {
+            SCP_2536_Controller.singleton.Cooldown = SCP_2536_Controller.singleton.CooldownAmount;
+            ReferenceHub referenceHub = player.ReferenceHub;
+
+            if (addToChosenList)
+            {
+                SCP_2536_Controller.singleton.PlayersAlreadyChosen.Add(referenceHub.characterClassManager.UserId);
+            }
+
+            RoomInformation.ZoneType currentZoneType = referenceHub.localCurrentRoomEffects.PlayersCurrentRoom.CurrentZoneType;
+            if (currentZoneType == RoomInformation.ZoneType.LCZ || currentZoneType == RoomInformation.ZoneType.ENTRANCE || currentZoneType == RoomInformation.ZoneType.HCZ)
+            {
+                SCP2536_Spawn_Location scp2536_Spawn_Location = null;
+                foreach (SCP2536_Spawn_Location scp2536_Spawn_Location2 in referenceHub.localCurrentRoomEffects.LastRoomIn.GetComponentsInChildren<SCP2536_Spawn_Location>())
+                {
+                    if (referenceHub.localCurrentRoomEffects.PlayersCurrentRoom.CurrentRoomType == RoomInformation.RoomType.HCZ_TESLA)
+                    {
+                        if (scp2536_Spawn_Location == null)
+                        {
+                            scp2536_Spawn_Location = scp2536_Spawn_Location2;
+                        }
+                        else if ((scp2536_Spawn_Location.transform.position - referenceHub.playerMovementSync.RealModelPosition).sqrMagnitude < (scp2536_Spawn_Location2.transform.position - referenceHub.playerMovementSync.RealModelPosition).sqrMagnitude)
+                        {
+                            scp2536_Spawn_Location = scp2536_Spawn_Location2;
+                        }
+                    }
+                    else if (scp2536_Spawn_Location == null)
+                    {
+                        scp2536_Spawn_Location = scp2536_Spawn_Location2;
+                    }
+                    else if ((scp2536_Spawn_Location.transform.position - referenceHub.playerMovementSync.RealModelPosition).sqrMagnitude > (scp2536_Spawn_Location2.transform.position - referenceHub.playerMovementSync.RealModelPosition).sqrMagnitude)
+                    {
+                        scp2536_Spawn_Location = scp2536_Spawn_Location2;
+                    }
+                }
+
+                scp2536_Spawn_Location.IsTreeActive = true;
+                scp2536_Spawn_Location.RpcSetTreeState(true);
+                SCP2536_Present[] componentsInChildren2 = scp2536_Spawn_Location.GetComponentsInChildren<SCP2536_Present>();
+                List<SCP_2536_Controller.Valid2536Scenario> list = SCP_2536_Controller.singleton.GetAllValid2536Scenarios(referenceHub).ToList<SCP_2536_Controller.Valid2536Scenario>();
```

Comment:
```suggestion
                List<SCP_2536_Controller.Valid2536Scenario> list = SCP_2536_Controller.singleton.GetAllValid2536Scenarios(referenceHub);
```

Here we don't need a ToList() call, the method creates one for us